package-lock.json

```diff
@@ -18,7 +18,7 @@
         },
         "@types/events": {
             "version": "1.2.0",
-            "resolved": "http://registry.npmjs.org/@types/events/-/events-1.2.0.tgz",
```

Comment:
What version of npm are you on? I'm on `6.4.1` and it doesn't make these changes...

Reply:
Well we're gonna have to figure this out on Monday otherwise it'll be super annoying if each of our machines changes this back and forth lol

Reply:
Well, we should be secure with our http protocol anyway 😜